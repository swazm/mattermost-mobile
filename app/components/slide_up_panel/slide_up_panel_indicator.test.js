// Copyright (c) 2015-present SWAZM, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import SlideUpPanelIndicator from './slide_up_panel_indicator';

describe('SlideUpPanelIndicator', () => {
    const baseProps = {
        dragIndicatorColor: '#fff',
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <SlideUpPanelIndicator {...baseProps}/>
        );

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
