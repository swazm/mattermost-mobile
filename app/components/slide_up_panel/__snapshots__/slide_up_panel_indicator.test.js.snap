// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlideUpPanelIndicator should match snapshot 1`] = `
<AnimatedComponent
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "marginVertical": 10,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 25,
        "height": 5,
        "opacity": 0.9,
        "width": 62.5,
      }
    }
  />
</AnimatedComponent>
`;
